README.md

```diff
@@ -24,6 +23,24 @@ Bonus if you can add filters!
 
 Remember to proceed in small steps! Getting confused? Make the steps even smaller.
 
+## User stories
```

Comment:
Outlining your identified user stories is very helpful. A next could be to add some comments about which have been achieved, how, and what blockers you expect in the ones that haven't been achieved